### ISSUE TITLE
Version 0.11.1 seems to be incompatible with Error Prone 2.20.0

### ISSUE THREAD

**Issue by pzygielo** [PicnicSupermarket/error-prone-support#686]
### Describe the bug

Similar to
- #622

but with
```
java.lang.AssertionError: java.lang.IllegalArgumentException: Cannot combine scanner suppliers with different implementations of 'MissingRefasterAnnotation': tech.picnic.errorprone.bugpatterns.MissingRefasterAnnotation, com.google.errorprone.bugpatterns.MissingRefasterAnnotation
```
instead of `StringCaseLocaleUsage` when upgrading EP from 2.19.1 to 2.20.0.

### Setup

- Operating system: ubuntu
- Java version: Java version: 17.0.7, vendor: Eclipse Adoptium
- Error Prone version: 2.20.0
- Error Prone Support version: 0.11.1

**Comment by Stephan202** [comment]
Hey @pzygielo! Indeed, this is due to another check that we upstreamed ("no good deed goes unpunished", haha). I filed #685 for this earlier today, and expect a release early next week.

**Comment by pzygielo** [comment]
For my excuse: I scanned only issues, and completely didn't think about PRs. So I missed the linked PR...

**Comment by Stephan202** [comment]
No worries at all! This ticket makes it easier for others to find the issue as well :+1:

**Comment by Stephan202** [comment]
Hey @pzygielo. Thanks for your patience; we just released an Error Prone 2.20.0-compatible version [0.12.0](https://github.com/PicnicSupermarket/error-prone-support/releases/tag/v0.12.0). Let us know if you hit any issues with this new release :)
